scripts/rotate_cloud_secrets.sh

```diff
@@ -0,0 +1,46 @@
+if brew ls --versions jq gh > /dev/null; then
+  continue
+else
+  echo "jq and gh packages must be installed. Please run 'brew install jq gh'"
+  exit 1
+fi
+
+if cf --version > /dev/null; then
+  continue
+else
+  echo "The cf CLI must be installed. Please see more information: https://docs.cloudfoundry.org/cf-cli/install-go-cli.html"
+  exit 1
+fi
+
+cf spaces
+read -p "Are you logged in to the dotgov-poc CF space above? (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    cf login -a https://api.fr.cloud.gov --sso
+fi
+
+gh auth status
+read -p "Are you logged into a Github account with access to cisagov/dotgov? (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    gh auth login
+fi
+
+echo "Great, removing and replacing Github CD account..."
+cf delete-service-key github-cd-account github-cd-key
+cf create-service-key github-cd-account github-cd-key
+cf service-key github-cd-account github-cd-key
+read -p "Please confirm we should set the above username and key to Github secrets. (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    exit 1
+fi
+
+cf service-key github-cd-account github-cd-key | sed 1,2d  | jq -r '[.username, .password]|@tsv' | 
```

Comment:
I had to change this `jq` command to get it to work. I figured it was a typo in the PR but could you double check on your end?

This is what the output of `cf` looks like for me:

```json

{
  "credentials": {
    "password": "hhhhhhhhhh",
    "username": "hhhhhhhhhh"
  }
}

```
```suggestion
cf service-key github-cd-account github-cd-key | sed 1,2d  | jq -r '.credentials|[.username, .password]|@tsv' | 
```

Reply:
Ahhh this might be a difference in versions of `cf`!

Reply:
What version of cf are you running?

Reply:
That was it! The version at 4d44c86 works fine when I use `cf version 7.5.0+0ad1d63.2022-06-04`.

Since this script will break on future version of `cf`, I'd recommend indicating the fix in a code comment to speed up future debugging.